### ISSUE TITLE
Just + babel and webpack config

### ISSUE THREAD

**Issue by jkovzels** [microsoft/just#292]
New to `just` and new to `babel`.

Then I started the project with `npm init uifabric` and in came with `bebelrc.js` in the package. Now I'm trying to add babel plugin and it seems like is babel is not there at all. Search by the repo gives very little results, I'm a missing something or I should add `babel-loader` to webpack config myself.

**Comment by ecraig12345** [comment]
@kenotron I agree this is weird--we should either get rid of the babelrc from the templates (and maybe add a separate babel-based template), or add babel as a dependency.
